src/sass/phone.scss

```diff
@@ -38,7 +38,6 @@
         margin-bottom: 0 !important;
 
         .consent-legal {
-          padding: 0;
           height: 70vh;
```

Comment:
Removed this to allow some padding between the modal text and edge of the modal at small screens.